Rubberduck.sln

```diff
@@ -149,8 +151,23 @@ Global
 		{AC4F1D22-D74B-45FF-AB0C-CC2A104FE023}.Release|x64.Build.0 = Release|Any CPU
 		{AC4F1D22-D74B-45FF-AB0C-CC2A104FE023}.Release|x86.ActiveCfg = Release|Any CPU
 		{AC4F1D22-D74B-45FF-AB0C-CC2A104FE023}.Release|x86.Build.0 = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|x64.Build.0 = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|x86.Build.0 = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|Any CPU.Build.0 = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|x64.ActiveCfg = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|x64.Build.0 = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|x86.ActiveCfg = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|x86.Build.0 = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {F927A957-7359-4572-94C7-2CFBE68C7334}
+	EndGlobalSection
```

Comment:
That's new. Why?

Reply:
Perhaps this snuck in before I nuked when I removed the VSTO project. Otherwise it is not (at least intentionally) mine.

Reply:
I have never poked around in the SLN files, so cannot definitively say that these changes are no the result of my work. I will try eliminating them and see what happens on my system.